mopidy_listenbrainz/frontend.py

```diff
@@ -15,14 +19,142 @@ class ListenbrainzFrontend(pykka.ThreadingActor, CoreListener):
     def __init__(self, config, core):
         super().__init__()
         self.config = config
+        self.library = core.library
+        self.playlists = core.playlists
+        self.playlists_update_timer = None
 
     def on_start(self):
         self.lb = Listenbrainz(
             self.config["listenbrainz"]["token"],
             self.config["listenbrainz"]["url"],
+            self.config["proxy"],
         )
         logger.debug("Listenbrainz token valid!")
 
+        if self.config["listenbrainz"].get("import_playlists", False):
+            search_schemes = self.config["listenbrainz"].get(
+                "search_schemes", "local:"
```

Comment:
This is a `list` by default, right?
```suggestion
                "search_schemes", ["local:"]
```